Showcase-iOS/Contact Us/Interactor/ContactUsInteractableImplementation.swift

```diff
@@ -5,25 +5,25 @@ import Firebase
 class ContactUsInteractableImplementation: ContactUsInteractable {
     
     var contactUsPresenter: ContactUsPresentable
-    var dataReference: DataReferenceable?
+    var dataReference: DataReferenceable
     
-    init(with contactUsPresenter: ContactUsPresentable,_ dataRefence: DataReferenceable?) {
-        self.dataReference = dataRefence
+    init(with contactUsPresenter: ContactUsPresentable,_ dataReference: DataReferenceable) {
+        self.dataReference = dataReference
         self.contactUsPresenter = contactUsPresenter
     }
     
     func retrieveContacts() {
         var offices: [Office] = [Office]()
-        guard let dataReference = dataReference else {return }
         let databaseReference = FirebaseRetrieverableImplementation(reference: dataReference)
         databaseReference.fetchFirebaseData(from: .office) { (snapshot, error) in
             if let error = error {
                 self.contactUsPresenter.onRetrieveOfficesFailed(with: error)
             } else if let result = snapshot as? DataSnapshot {
-                offices = result.children.map({ Office(with: $0 as? DataSnapshot)})
+                offices = result.children.map({ Office(with: ($0 as! [String: Any]))})
```

Comment:
We need to not explicity unwrap anything. Its a rule of thumb. When i add swiftLint this will be a rule that you cant do this. So lets not do it. creates bugs

Reply:
Cool